# This is a BitKeeper generated patch for the following project:
# Project Name: Linux kernel tree
# This patch format is intended for GNU patch command version 2.5 or higher.
# This patch includes the following deltas:
#	           ChangeSet	1.1369  -> 1.1370
#	drivers/mtd/cmdlinepart.c	1.5     -> 1.6
#
# The following is the BitKeeper ChangeSet Log
# --------------------------------------------
# 03/10/16	noah@caltech.edu	1.1370
# Remove needless include of asm/setup.h in drivers/mtd/cmdlinepart.c.  This
# was breaking compilation on architectures that lack such a header.
# --------------------------------------------
#
diff -Nru a/drivers/mtd/cmdlinepart.c b/drivers/mtd/cmdlinepart.c
--- a/drivers/mtd/cmdlinepart.c	Fri Oct 17 13:40:56 2003
+++ b/drivers/mtd/cmdlinepart.c	Fri Oct 17 13:40:56 2003
@@ -28,7 +28,6 @@

 #include <linux/mtd/mtd.h>
 #include <linux/mtd/partitions.h>
-#include <asm/setup.h>
 #include <linux/bootmem.h>

 /* error message prefix */
